### DISCUSSION TITLE
Support 2/4 time signature in editor timing

### DISCUSSION THREAD

**Discussion by sr229** [ppy/osu#13345] [Ideas] [answered]
On certain songs, such as [Bossa Nova](https://www.bbc.co.uk/bitesize/guides/zg4hb82/revision/5#:~:text=Bossa%20Nova,on%20melody%20than%20rhythmic%20percussion.), you would need 2/4 to accurately map the song's rhythm. Currently in stable, this is not possible via GUI, making certain songs unsupported for mapping.

**Comment by gagahpangeran** [comment, marked as answer]
I think this is in scope of https://github.com/ppy/osu/issues/2316

**Reply by sr229** [reply]
Yep, this is within scope. I'm yet to discover more unusual time signatures but this one seems to be the most outlier since it uses 2/4 (very uncommon, though that's an understatement considering there's a lot of Bossa Nova tracks out there).
